docs/sources/fundamentals/dashboards-overview/index.md

```diff
@@ -38,37 +38,37 @@ A Grafana plugin is software that adds new capabilities to Grafana. They come in
 
```

Comment:
On the line above (line 37), I'm not clear what the two data models are in the phrase "reconcile the differences in data models". The data model of the data source and the dashboard? Or something else?

Reply:
@imatwawana this relates to your last point -- let me sort of explain, then you be the judge of how best to express this.

So there's a zillion different databases.  Some have json documents, some have tables, some have text, some have metrics, lots of "different kinds of stuff"

In grafana, dashboard panels only understand one thing:  data frames.

So what I'm trying to say here is that plugins know how to talk to underlying databases in whatever their lingua franca is (tables, json, text, metrics, whatever) -- and translate that into "data frames".  In this way, we can use the same visualization panel (like a timeseries viz) on *any kind of data*.

That's a longer version of what I'm trying to say in that paragraph, but you be the judge of what best arrives.

Reply:
Okay that's much clearer to me. I've made a suggestion above.